internal/services/sql/sql_managed_instance_failover_group_resource_test.go

```diff
@@ -0,0 +1,177 @@
+package sql_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/sql/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type SqlInstanceFailoverGroupResource struct{}
+
+func TestAccAzureRMSqlInstanceFailoverGroup_basicManual(t *testing.T) {
```

Comment:
We should throw in another test where we add/remove another sql managed instance and mess with some of the other attributes. The time it'll take to spin that up is 😬  but we should confirm it works

Reply:
I've setup one extra test, switching to another secondary (limit is 1 at the moment). Besides that I've added one more step to the basic test. 

If you want more, let me know 😬